classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
```

Comment:
Yeah, so I think this is the largest chunk that gets passed through `Capsule::raw`:

```php
$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
```

When I take that out of the `Capsule::raw` it tells me:

```
<h1>DB Error: Unknown column 'section_title' in 'on clause'</h1>
```

Maybe you or @kaschioudi have an idea for getting around this, but it's not entirely clear to me what the issue is, or why wrapping it in `Capsule::raw` solves the problem.